editor/src/components/canvas/canvas-strategies/absolute-move-strategy.spec.browser2.tsx

```diff
@@ -0,0 +1,373 @@
+import {
+  getPrintedUiJsCode,
+  makeTestProjectCodeWithSnippet,
+  renderTestEditorWithCode,
+  TestScenePath,
+} from '../ui-jsx.test-utils'
+import { act, fireEvent } from '@testing-library/react'
+import * as EP from '../../../core/shared/element-path'
+import { selectComponents } from '../../editor/actions/action-creators'
+import { CanvasControlsContainerID } from '../controls/new-canvas-controls'
+import CanvasActions from '../canvas-actions'
+import { createInteractionViaMouse } from './interaction-state'
+import { zeroCanvasPoint } from '../../../core/shared/math-utils'
+import { emptyModifiers } from '../../../utils/modifiers'
+
+describe('Absolute Move Strategy', () => {
+  it('moves absolute positioned element', async () => {
+    const renderResult = await renderTestEditorWithCode(
+      makeTestProjectCodeWithSnippet(`
+        <div style={{ width: '100%', height: '100%' }} data-uid='aaa'>
+          <div
+            style={{ backgroundColor: '#0091FFAA', position: 'absolute', left: 40, top: 50, width: 200, height: 120 }}
+            data-uid='bbb'
+            data-testid='bbb'
+          />
+        </div>
+      `),
+      'await-first-dom-report',
+    )
+
+    const areaControl = renderResult.renderedDOM.getByTestId('bbb')
+    const areaControlBounds = areaControl.getBoundingClientRect()
+    const canvasControlsLayer = renderResult.renderedDOM.getByTestId(CanvasControlsContainerID)
+
+    fireEvent(
+      canvasControlsLayer,
+      new MouseEvent('mousedown', {
+        bubbles: true,
+        cancelable: true,
+        metaKey: true,
+        clientX: areaControlBounds.left + 5,
+        clientY: areaControlBounds.top + 5,
+        buttons: 1,
+      }),
+    )
+
+    await act(async () => {
+      const dispatchDone = renderResult.getDispatchFollowUpActionsFinished()
+      fireEvent(
+        canvasControlsLayer,
+        new MouseEvent('mousemove', {
+          bubbles: true,
+          cancelable: true,
+          metaKey: false,
+          clientX: areaControlBounds.left + 45,
+          clientY: areaControlBounds.top - 20,
+          buttons: 1,
+        }),
+      )
+      await dispatchDone
+    })
+
+    await act(async () => {
```

Comment:
I remember we had these "multiple acts below each other, dispatching the same mouse move" in old tests, because otherwise the dom-walker was just inconsistent. Have you tried with a single mouse move event? it _should_ be much better nowadays...

Reply:
I removed those extra mouse moves, it seems okay now